setup.cfg

```diff
@@ -130,8 +130,6 @@ release =
 dev =
     black
     check-manifest>=0.42
-    flake8
-    isort
```

Comment:
Why isn't `ruff` part of the dependencies, in place of those two?

Reply:
ruff updates comes very often. So I prefer to rely on pre-commit version pinning,

Reply:
Ah, so `pre-commit install` will handle ruff then?

Reply:
`pre-commit run`

Reply:
right, but i mean the installation of ruff—I assume i still need it in my env for it to work.

Reply:
No. Pre-commit handles the installation of hooks independly of your rev environment.

Reply:
I guess this means that I cannot manually call it from a new env? But I guess that's fine, I never do that :P

Reply:
It just means that it will not be automatically installed. You could still install them manually.

Reply:
of course :P